ocs_ci/utility/operators.py

```
@@ -275,10 +275,50 @@ def _customize_operatorgroup(self, operatorgroup_data: dict):
+        the OperatorGroup automatically.
+
+        Returns:
+            bool: True if at least one OperatorGroup exists, False otherwise
+        """
+        from ocs_ci.utility.utils import exec_cmd
+
+        cmd = (
+            f"oc get operatorgroup -n {self.namespace} --no-headers 2>/dev/null | wc -l"
+        )
+        try:
+            result = exec_cmd(cmd, shell=True)
```

Ruff (0.15.13)

[error] Function call with `shell=True` parameter identified, security issue

(S604)